system/handlers/admin/DataManager.cfc

```diff
@@ -198,6 +198,7 @@
 			var filterValue    = "";
 			var orderBy        = rc.orderBy       ?: "label";
 			var labelRenderer  = rc.labelRenderer ?: "";
+			var useCache	   = structKeyExists(rc, 'useCache') ? rc.useCache : true;
```

Comment:
FYI: Tweaking this to the more concise:

```
var useCache = IsTrue( rc.useCache ?: "" );
```

`IsTrue()` is a coldbox helper function defined in Preside core so available to handlers and views. It ensures that the result is boolean, no matter the user input.